runtime/interpreter/encoding_benchmark_test.go

```diff
@@ -16,146 +16,149 @@
-
-			b.ReportAllocs()
-			b.ResetTimer()
-
-			for i := 0; i < b.N; i++ {
-				_, _, err := EncodeValue(bm.value, nil, true, nil)
-				require.NoError(b, err)
-			}
-		})
-	}
-}
-
-func BenchmarkDecodeCBOR(b *testing.B) {
-	benchmarks := prepareCBORTestData()
-
-	for _, bm := range benchmarks {
-		b.Run(bm.name, func(b *testing.B) {
-
-			encoded, _, err := EncodeValue(bm.value, nil, true, nil)
-			require.NoError(b, err)
-
-			b.ReportAllocs()
-			b.ResetTimer()
-
-			for i := 0; i < b.N; i++ {
-				_, err = DecodeValue(encoded, nil, nil, CurrentEncodingVersion, nil)
-				require.NoError(b, err)
-			}
-		})
-	}
-}
-
-// prepareCBORTestData processes testdata/*.cbor.gz files and
-// returns a []benchmark including each benchmark name and Value.
-// For safety, the number of *.cbor.gz files processed is limited and
-// io.LimitReader is used to limit uncompressed CBOR data size.
-func prepareCBORTestData() []benchmark {
-	// maxNumFiles limits the number of files processed in testdata folder.
-	const maxNumFiles = 100
-
-	// maxCBORSize limits max size of each uncompressed CBOR data to 16 MB.
-	const maxCBORSize = 16 * 1024 * 1024
-
-	// Compile regular expression to match version number in file name
-	versionRegExp := regexp.MustCompile(`_v([\d]+)_`)
-
-	fileNames, err := filepath.Glob("testdata/*.cbor.gz")
-	if err != nil {
-		panic(err)
-	}
-
-	numFiles := len(fileNames)
-	if numFiles > maxNumFiles {
-		numFiles = maxNumFiles
-
-		fmt.Printf("Found %d *.cbor.gz files in testdata.  Processing only the first %d files.\n",
-			len(fileNames),
-			maxNumFiles)
-	}
-
-	var benchmarks []benchmark
-	for _, fileName := range fileNames[:numFiles] {
-
-		// Read cbor.gz file
-		f, err := os.Open(fileName)
-		if err != nil {
-			panic(err)
-		}
-		defer f.Close()
-
-		zr, err := gzip.NewReader(f)
-		if err != nil {
-			panic(err)
-		}
-		defer zr.Close()
-
-		lzr := io.LimitReader(zr, maxCBORSize)
-
-		var buf bytes.Buffer
-		fileSize, err := io.Copy(&buf, lzr)
-		if err != nil {
-			panic(err)
-		}
-
-		// File name without path nor extension
-		name := strings.TrimSuffix(filepath.Base(fileName), ".cbor.gz")
-
-		// Get version number from file name
-		// If file name doesn't match "_v([\d+])_", version number is default to current version.
-		version := CurrentEncodingVersion
-		m := versionRegExp.FindStringSubmatch(name)
-		if len(m) > 1 {
-			v, err := strconv.ParseUint(m[1], 10, 16)
-			if err == nil {
-				version = uint16(v)
-			}
-		}
-
-		// Construct benchmark name
-		name += fmt.Sprintf("_%dbytes", fileSize)
-
-		// Decode test data to value
-		owner := common.BytesToAddress([]byte{})
-
-		value, err := DecodeValue(buf.Bytes(), &owner, nil, version, nil)
-		if err != nil {
-			panic(fmt.Sprintf("failed to decode value in file %s: %s\n", fileName, err))
-		}
-
-		// Add to benchmarks
-		benchmarks = append(benchmarks, benchmark{name: name, value: value})
-	}
-
-	return benchmarks
-}
+package interpreter_test
```

Comment:
Benchmarks are now outdated. Should we just remove it? Or remove the atree-based values, and keep the non-atree based values?